app/controllers/admin/dashboard_controller.rb

```diff
@@ -34,6 +35,26 @@ def prep_projects_grid
     @status_filter_options = STATUS_FILTERS.map { |f| [I18n.t("dashboard.status_options.#{f}"), f] }
   end
 
+  # Prepare grids for all users inside selected division
+  def prep_projects_grids
+    @people = @division.people.where(has_system_access: true).where.not(id: @person.id)
+
+    @people_with_projects = []
+    @grids = {}
+    @people.each do |person|
+      if person.agent_projects.length > 0
+        @people_with_projects << person
+        @grids["person_#{person.id}"] = initialize_grid(
+          person.agent_projects.order(created_at: :desc, updated_at: :desc).limit(15),
+          include: [:primary_agent, :secondary_agent],
+          per_page: 5,
```

Comment:
Limits were added for performance reasons, but it seems this is loading fast enough. So I agree. Have removed the earlier limits.